### PR TITLE
feat: configuration through node-config

### PR DESCRIPTION
**Pull request by ironFe93**
## What

Use node-config instead of Configurator.ts.
## Why

So that we don't have to pass around a config object.

## Refs

- (optional: include links to other issues, PRs, tickets, etc)

### CODE REVIEW

**Review by jurajpiar — CHANGES_REQUESTED** [review]
replace `src/Configurator.ts` with `config/default.json`.

File: src/Configurator.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        methodSuffix,
+        jsonStringifyRequest
+    };
+
+    return {
+        ...defaultEnvelopingConfig,
+        ...partialConfig,
+        ...resolvedConfig
+    } as EnvelopingConfig;
+}
```
[the remark this answers is not in the document]
Comment (ironFe93) [reply src/Configurator.ts:63]: so it is even simpler than this

[the remark this answers is not in the document]
Comment (ironFe93) [reply src/Configurator.ts]: removed

**Review by jurajpiar — APPROVED** [review]

**Comment by jurajpiar** [comment]
clean your history before merging please once the ci passes

**Review by franciscotobar — APPROVED** [review]
LGTM

**Pull request merged**